designsafe/apps/api/messages/models.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals
+import logging
+from django.db import models
+
+logger = logging.getLogger(__name__)
+TYPE_CHOICES = (
+    ('alert-warning', 'WARNING'),
+    ('alert-danger', 'DANGER'),
+    ('alert-info', 'INFO'),
+    ('alert-success', 'SUCCESS'),
+    )
+
+class PageAlert(models.Model):
```

Comment:
@rstijerina Do we want to add a `is_active` attribute to keep a history of past alerts and not having to delete them if we don't want to show them?

Reply:
Yeah that might be a good idea, especially if we find we have a few standard alerts that we'll need to activate periodically for hpc downtimes, etc.